### PR TITLE
Refactored output directory structure

### PR DESCRIPTION
**Pull request by tarunbhardwaj**

### CODE REVIEW

**Comment by prakashpp** [comment]
LGTM

File: pelicanconf.py [hunk 1]
```diff
@@ -68,3 +68,18 @@
...
+
+# Output dir structure
+ARTICLES_URL_ROOT = 'blogs'
+
+INDEX_SAVE_AS = '%s/index.html' % ARTICLES_URL_ROOT
+ARTICLE_URL = "%s/{slug}.html" % ARTICLES_URL_ROOT
+ARTICLE_SAVE_AS = "%s/{slug}.html" % ARTICLES_URL_ROOT
+
+CATEGORIES_SAVE_AS = '%s/categories.html' % ARTICLES_URL_ROOT
+CATEGORY_URL = "%s/categor/{slug}/" % ARTICLES_URL_ROOT
```
Comment (sharoonthomas) [inline pelicanconf.py]: category

File: pelicanconf.py [hunk 2]
```diff
@@ -68,3 +68,18 @@
...
+CATEGORY_SAVE_AS = "%s/categor/{slug}/index.html" % ARTICLES_URL_ROOT
```
Comment (sharoonthomas) [inline pelicanconf.py]: category

**Comment by sharoonthomas** [comment]
the URL's are now absolute right ? Do you think this will affect language specific sites and pages ? could you generate once and check ?

**Pull request closed without merging**